storage/innobase/handler/ha_innodb.cc

```diff
@@ -1938,6 +1938,7 @@ trx_is_registered_for_2pc(
 
 /*********************************************************************//**
 Note that a transaction owns the prepare_commit_mutex. */
+__attribute__((__unused__))
```

Comment:
Is this also used in some configs only?

Reply:
Yes, only if EXTENDED_FOR_COMMIT_ORDERED

Reply:
OK, let's use it then. (I suspect that this define is ours, not upstream, and we could remove it, but 5.5 is a mature release next in line for EOL, so maybe let's just fix the warning)